coding_systems/versioning/models.py

```diff
@@ -26,10 +26,11 @@ class Meta:
         unique_together = ("coding_system", "release_name", "valid_from")
 
     def save(self, *args, **kwargs):
-        if not self.database_alias:
-            self.database_alias = slugify(
-                f"{self.coding_system}_{self.release_name}_{self.valid_from.strftime('%Y%m%d')}"
-            )
+        # CodingSystem methods and the database router depend on the format
+        # of database_alias following an expected pattern, therefore we overwrite any
+        # attempts to manually set it.
```

Comment:
The approach taken elsewhere is to only make changes through functions in actions.py.  This isn't enforced though, so perhaps it's right to be cautious here.

Or would it be better to assert that if `self.database_alias` is not `None`, then it's as expected?  That way, we can catch any code that's doing the wrong thing.

Reply:
Hmm, maybe asserting the format is correct is better than silently overwriting it. It _should_ only be set by the import_data function for each coding system (which will use a common function in the next PR), but that's also not enforced